src/buffer/out/TextColor.h

```diff
@@ -133,11 +134,10 @@ struct TextColor
     COLORREF GetColor(const std::array<COLORREF, TABLE_SIZE>& colorTable, const size_t defaultIndex, bool brighten = false) const noexcept;
     BYTE GetLegacyIndex(const BYTE defaultIndex) const noexcept;
 
-    constexpr BYTE GetIndex() const noexcept
-    {
-        return _index;
-    }
-
+    BYTE GetIndex() const noexcept;

```

Comment:
dumb question - does moving this out of the header harm the inlineability of it? does LTCG largely help with that?

Reply:
`inline` isn't a hint for inlining anymore, so it shouldn't make any difference. LTCG makes no difference either I think, because if anything it's probably inlined during initial compilation already due to its small size.

Reply:
Ah, I don't mean because of the `inline` keyword. Method bodies in header files _can_ be inlined automatically, by the compiler, without the linker's help (LTCG). If you move the body into the cpp file, it _can't_ be inlined during initial compilation because the compiler can only see the body from a single translation unit. It would require LTCG to come through and inline it at the call sites, now, because the other translation units couldn't see the body at compile time.

Reply:
Ah good point. This would be a good reason to move it back to the header.